marimo/_runtime/output/_output.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2023 Marimo. All rights reserved.
+import sys
+
+from marimo._ast.cell import CellId_t
+from marimo._messaging.ops import CellOp
+from marimo._output import formatting
+from marimo._output.rich_help import mddoc
+from marimo._plugins.stateless.flex import vstack
+from marimo._runtime.context import get_context
+
+
+def write_internal(cell_id: CellId_t, value: object) -> None:
+    output = formatting.try_format(value)
+    if output.traceback is not None:
+        sys.stderr.write(output.traceback)
+    CellOp.broadcast_output(
+        channel="output",
+        mimetype=output.mimetype,
+        data=output.data,
+        cell_id=cell_id,
+        status=None,
+    )
+
+
+@mddoc
+def replace(value: object) -> None:
```

Comment:
Thoughts on naming?

I know we talked about `mo.output()` and `mo.output.append()`, but I thought this was clearer. I also considered `write` instead of `replace`, but opted for `replace` because it's more descriptive.

Reply:
i like the naming that you landed on - also reflecting on the `mo.output()` `mo.output.append()`, some people get confused (and so does IDE docs) when having something callback and has further properties